token/core/zkatdlog/crypto/common/array.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const Seperator = "%%"
+
```

Comment:
Not related to this PR, but, FYI - it's easier to just have a definition like:

```
type G1Array []*math.G1`
```

This makes it easier to initialize and it can be easily converted from and to `[]*math.G1`.

Reply:
I will address this in a PR dedicated to cleaning up the code.